src/main/java/uk/gov/hmcts/reform/finrem/caseorchestration/model/ccd/wrapper/GeneralEmailWrapper.java

```diff
@@ -22,6 +22,7 @@ public class GeneralEmailWrapper implements HasCaseDocument {
     private String generalEmailRecipient;
     private String generalEmailCreatedBy;
     private String generalEmailBody;
+    @JsonInclude(JsonInclude.Include.ALWAYS)
```

Comment:
OK.  So the migration will leave the generalEmailUploadedDocument property in the stored JSON, but with the value explicitly set to null. 

So your testing has confirmed that still works for users. Can you create a follow up Jira tick post migration to revert this.  Just so we don't leave all the temp fields behind for normal events.